src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py

```diff
@@ -21,29 +24,39 @@ def dummy_data():
     return r, v, t, m, start_lambda, end_lambda, step_size
 
 
-def test_staticgeodesicplotter_has_axes(dummy_data):
-    r, v, _, m, _, el, ss = dummy_data
+def test_staticgeodesicplotter_has_axes(create_test_data):
+    """
+    Function to test if the plot has axes
+    """
+    r, v, _, m, _, el, ss = create_test_data
     cl = StaticGeodesicPlotter(m)
     assert isinstance(cl.ax, mpl.axes.SubplotBase)
     assert cl.time.value == 0.0
     assert cl._attractor_present is False
 
 
 @mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.plot_attractor")
-def test_plot_calls_plot_attractor(mock_plot_attractor):
+def test_plot_calls_plot_attractor1(mock_plot_attractor):
+    """
+    Function to test the plot_attractor function
+    """
     r = [306 * u.m, np.pi / 2 * u.rad, np.pi / 2 * u.rad]
     v = [0 * u.m / u.s, 0 * u.rad / u.s, 951.0 * u.rad / u.s]
     m = 4e24 * u.kg
     el = 0.002
     ss = 0.5e-6
     cl = StaticGeodesicPlotter(m)
     cl.plot(r, v, el, ss)
+    cl.show()
     mock_plot_attractor.assert_called_with()
     assert cl._attractor_present
 
 
-@mock.patch("einsteinpy.plotting.geodesics_static.plt.savefig")
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.save")
```

Comment:
Why did you change this?

Reply:
Oh, this. I thought since some others are starting from StaticGeodesicPlotter, there should be uniformity?

Reply:
Read about what mocking is :) 
https://docs.python.org/3/library/unittest.mock.html